script/QA/QA_20.txt

```diff
@@ -121,72 +127,81 @@ Correct. Depending on the player's perspective,
 [sha:585290c28b9ed6cd1b620209c1ec8d9e20fe0aa4]{
 -- Page 179, Offset 43647.
 -- 幸福なのかに別れる、メリーバッドエンドです。
-this could either be a sad or happy ending. A merry bad end.
+ this could%{n}
+be a sad ending or a happy ending. A merry bad end.
```

Comment:
I almost feel like 
>A Merry Bad End.
this makes it sound like its some kind of category of bad ends, which is funny. She also has another dojo where she calls one of the endings a "World End" instead of a "Bad End" so maybe she just like coming up with these.